### PR TITLE
Fix outdated action tooltip on changing plugin setting

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

**Comment by nyalldawson** [comment]
Looks like this was filed as #17

**Comment by borysiasty** [comment]
@nyalldawson thanks fot this gift! I was planning to get down to it tomorrow :)

File: reloader_plugin.py [hunk 1]
```diff
@@ -106,14 +106,14 @@ def theBestShortcutForPluginReload(self):
...
+      "Reload chosen plugin",
       self.iface.mainWindow()
     )
-    self.actionRun.setWhatsThis(u"Reload chosen plugin")
+    self.actionRun.setToolTip("Reload chosen plugin")
     plugin = currentPlugin()
     if plugin:
-      self.actionRun.setWhatsThis(u"Reload plugin: %s" % plugin)
-      self.actionRun.setText(u"Reload plugin: %s" % plugin)
+      self.actionRun.setToolTip("Reload plugin: {}".format(plugin))
```
Comment (borysiasty) [inline reloader_plugin.py:115]: Do you know when the good old formatting is going to be deprecated?

Comment (nyalldawson) [reply reloader_plugin.py:115]: No idea, sorry...

Comment (raymondnijssen) [reply reloader_plugin.py:115]: Thank you! :*

**Pull request merged**